frontend/src/pages/WhatIsHealthEquity/News/SinglePost.tsx

```diff
@@ -119,7 +119,7 @@ export default function SinglePost() {
             )}
             {!isLoading && !error && (
               <img
-                src={articleImage}
+                src={articleImage || hetLogo}
                 className={styles.SingleArticleHeaderImg || hetLogo}
```

Comment:
oh it's on the next line down lol. so i tried to do the backup but just added the fallback to the className hahaha oops